src/ring_theory/subring.lean

```diff
@@ -507,7 +509,20 @@ instance : comm_ring (center R) :=
 
 end
 
-/-! # subring closure of a subset -/
+section division_ring
+
+variables {K : Type u} [division_ring K]
+
+instance : field (center K) :=
+{ inv := λ a, ⟨a⁻¹, set.inv_mem_center' a.prop⟩,
```

Comment:
Can you add `coe_inv` and `coe_div` simp lemmas about this definition?

Reply:
It would probably be good to provide an explicit `div` too

Reply:
Should there be `set.div_mem_center` as well then?

Reply:
I'll add `set.div_mem_center` anyways. Is there a reason to prefer having an explicit `div`? Since this means that I'll have to prove `div_eq_mul_inv` if I use the division from the division ring.

Reply:
Yes, the explicit div makes `coe_div` true by rfl which is often useful. The proof of `div_eq_mul_inv` should be trivial, likely `subtype.ext (div_eq_mul_inv _ _)`